source/_docs/configuration/securing.markdown

```diff
@@ -1,29 +1,79 @@
-To expose your instance to the internet, use a [VPN](https://pivpn.dev/), or an [SSH tunnel](/blog/2017/11/02/secure-shell-tunnel/). Make sure to expose the used port in your router. 
+| issue type | Solution|
+|---|---|
+| Physical damage | [Replace hardware](https://www.home-assistant.io/installation/), [restore a backup](https://www.home-assistant.io/common-tasks/os/#restoring-a-snapshot-on-a-new-install). |
+| Configuration issue | [restore a backup](https://www.home-assistant.io/common-tasks/os/#restoring-a-snapshot-on-a-new-install).|
+
+### Data Leakage Recovery
+
+| issue type | Solution|
+|---|---|
+| Cloud Service Data Breach | Change Cloud Service authentication. |
+| Local Home Assistant Data Breach | [Change Home Assistant authentication](https://www.home-assistant.io/docs/locked_out/#home-assistant-including-supervised) and  [restore a backup](https://www.home-assistant.io/common-tasks/os/#restoring-a-snapshot-on-a-new-install). |
+
+### Off-Site backups
+
+On-site/on-device backups are fast, cheap and easy, but in the event of physical damage to your hardware such as natural disaster, you may not have access to any on-site data.  It is recommended to store, at least some, backups off-site.  You can choose USB sticks, hard drives, a cloud provider, multiple providers, or any method to physically transport the data to a separate and secure location.  Any and all methods of storage can be used in your Backup Rotation Scheme. You can find multiple add-ons to [assist with off-site backup in the forums](https://community.home-assistant.io/search?q=tag%3Ahassio-addon%20backup%20).
+
+### Backup Rotation Scheme
+
+Nobody has unlimited storage. Disks can fill up, and cloud storage can be expensive. Because Home Assistant is intended to be the only solution you will ever need, eventually you will run out of space. The solution to balancing the storage requirements and duration of backups is a proper [Backup Rotation Scheme](https://en.wikipedia.org/wiki/Backup_rotation_scheme). A good rotation scheme, like [daily, weekly, monthly](https://en.wikipedia.org/wiki/Backup_rotation_scheme#Grandfather-father-son), will ensure you have access to the latest backups and you are able to locate a backup prior to any issue you may encounter.
+
+## Remote Access
 
-### Extras for manual installations
+If you want secure remote access, the easiest option is to use [Home Assistant cloud](/cloud/) by which you also [support](https://www.nabucasa.com/about) the founders of Home Assistant.
 
-As well as the above we advise that you consider the following to improve security:
+To expose your instance to the internet, use a [VPN](https://pivpn.dev/), or an [SSH tunnel](/blog/2017/11/02/secure-shell-tunnel/). Make sure to expose the used port in your router. Other options are to use TLS/SSL via the add-ons [Duck DNS integrating Let's Encrypt](/integrations/duckdns/) or [Let’s Encrypt](/docs/ecosystem/certificates/lets_encrypt/).
```

Comment:
I'm not sure if we should recommend anything at this point. 

`/docs/ecosystem/certificates/lets_encrypt/` -> Does not exist in our docs anymore and should've removed.

Reply:
Duck DNS is still supported.  I'd say it would be better to leave it because otherwise we are recommending Nabu Casa and nothing else, while the first paragraph says it doesn't depend on a cloud service.  I kinda like the idea of leaving this as-is because it shows the official Nabu Casa way and then the community way since the Let's Encrypt link directs to community.

Reply:
We do not support community methods and should therefor not be in our documentation.